src/test/operators/join_equi_test.cpp

```diff
@@ -67,7 +67,8 @@ TYPED_TEST(JoinEquiTest, InnerJoinIntFloatRadixBit) {
     join->execute();
     EXPECT_TABLE_EQ_UNORDERED(join->get_output(), expected_result);
 
-    for (size_t radix_bits : {1, 2, 3, 10, 17}) {
+    for (size_t radix_bits : {1, 2, 8}) {
+      std::cout << radix_bits << std::endl;
```

Comment:
?

Reply:
The test took 1s in release. We built 2^17 partitions.
2^8 is sufficient to check whether we can deal with #partitions>#rows.

Shall I add a comment?

Reply:
That explains the reduced test size, but what about the cout?

Reply:
Narf. Didn't see that. Was just there for debugging.

Will add the comment and remove the cout.

Reply:
Done.